src/rules/no-collapsible-if.ts

```diff
@@ -27,6 +27,12 @@ import { report, issueLocation } from '../utils/locations';
 const rule: Rule.RuleModule = {
   meta: {
     type: 'suggestion',
+    docs: {
+      description: 'Collapsible "if" statements should be merged',
+      category: 'Stylistic Issues',
```

Comment:
I would replace all `Stylistic Issues` rules in this PR with `Best Practices`. Stylistic Issues seems to be more about formatting.
We already have `type` property, so I would map `suggestion` to `Best Practices` and `problem` to `Possible Errors`

Reply:
👍 I like this mapping you are suggesting.